stylesheets/commons/DeckList.scss

```diff
@@ -3,7 +3,7 @@ Template(s): Artifact decks background
 Author(s): Ricci
 *******************************************************************************/
 
-.artifact-deck-#000000 {
+.artifact-deck-000000 {
```

Comment:
do we need to adjust something on the wiki for this?

Reply:
Doesn't seem to be used (but I don't trust the search on small wikis anymore), so I don't want to rmeove it entirely just yet

https://liquipedia.net/artifact/index.php?title=Special:Search&profile=all&search=insource%3A%22artifact-deck%22&fulltext=1

But sooner or later we need to go through and clean up unused classes